src/sql/workbench/common/editor/query/queryPlan2State.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+
+export class QueryPlan2State {
```

Comment:
Given that you're renaming this to ExecutionPlan everywhere else I would definitely suggest you clean up any references to query plan as a follow up, otherwise it's going to be very confusing for people going through this later on.

Reply:
That would be a very big change for this PR. I will create a new PR where I am renaming everything in QueryPlan2 to Execution Plan

Reply:
Agreed, sounds good